core/trading_engine.py

```
@@ -673,61 +960,164 @@ async def _execute_signals(self, entry_signals: List[Signal], exit_signals: List
+
+                # 주문 생성 - OrderRequest 사용 (테스트 코드와 동일)
+                order_request = OrderRequest(
+                    symbol=signal.symbol,
+                    side=OrderSide.BUY if signal.type == SignalType.BUY else OrderSide.SELL,
+                    order_type=OrderType.MARKET,
+                    quantity=quantity
                 )
+                logger.info(f"[DEBUG] 주문 객체 생성 완료 - symbol: {order_request.symbol}, side: {order_request.side.value}, quantity: {order_request.quantity}")
+
+                # 주문 실행 (place_order는 동기 함수)
+                logger.info(f"[DEBUG] place_order 호출 시작")
```

Ruff (0.14.1)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)